server/src/utils/regexp/file.regexp.ts

```diff
@@ -1 +1 @@
-export const allowedTypeRegExp = /image/g;
\ No newline at end of file
+export const allowedTypeRegExp = /image/;
```

Comment:
It's neccessary because of JavaScript [RegExp](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp) objects are stateful